res/css/views/rooms/_EventTile.scss

```diff
@@ -850,6 +860,10 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
 
         &:not([data-layout=bubble]) {
             padding-top: $spacing-16;
+
+            &:hover .mx_EventTile_line {
+                box-shadow: none; // don't show the verification left stroke in a thread (mx_ThreadView)
+            }
```

Comment:
it should be shown within ThreadView (a thread) just not the list of threads

Reply:
It is addressed with https://github.com/matrix-org/matrix-react-sdk/pull/8525/commits/ee487653e5cbaf7582c575d820955d023a2e3622.

![after](https://user-images.githubusercontent.com/3362943/167656534-2bd3b8c2-3558-4e2a-bf19-e7fa4aade323.png)